src/views/homepage/components/certifications-section/components/certifications-text-and-image/index.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from 'next/image'
+import svgHeroImage from './assets/certification-hero-image-outlined.svg'
+import StandaloneLink from 'components/standalone-link'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import { CertificationsTextAndImageProps } from './types'
+import { SplitLayoutWithBackground } from '../split-layout-with-background'
+import s from './certifications-text-and-image.module.css'
+
+/**
+ * Render a heading, description, and link with a certifications-themed
+ * background and accompanying presentation visual.
+ */
+export function CertificationsTextAndImage({
```

Comment:
`CertificationsTextAndImage` renders the bulk of the work here, being everything except the already-built cards grid component.